renderdoc/driver/vulkan/vk_layer.cpp

```diff
@@ -163,7 +163,7 @@ VkResult VKAPI_CALL hooked_vkCreateInstance(const VkInstanceCreateInfo *pCreateI
                                             const VkAllocationCallbacks *pAllocator,
                                             VkInstance *pInstance)
 {
-  WrappedVulkan *core = new WrappedVulkan("");
+  WrappedVulkan *core = new WrappedVulkan(0);
```

Comment:
What's the reasoning behind this change? The filename parameter here is only used on replay so it should probably be harmless (hooked_vkCreateInstance shouldn't be called on replay), but I'm not sure I understand it.